elm-format-lib/src/Parse/Literal.hs

```diff
@@ -126,7 +126,8 @@ chr =
     toError e = newErrorUnknown ("Error parsing char: " ++ show e)
   in
   do  s <- character toExpecation toError
-      processAs charLiteral $ sandwich '\'' (ES.toChars s)
+      case ES.toChars s of
+        [ c ] -> return c
```

Comment:
We should add a "please report ..." error for the fallback branch.

Reply:
Error message added in 8dca014